datafusion/common/src/config.rs

```diff
@@ -1696,6 +1697,13 @@ impl CsvOptions {
         self
     }
 
+    /// The character that terminates a row.
+    /// - default to '\n'
```

Comment:
If it's None as stated above -- won't it be CRLF by default (which is slightly more than `\n` according to [csv_core](https://docs.rs/csv-core/latest/csv_core/enum.Terminator.html) docs used internally by arrow-csv)?

Reply:
You're right. I missed modifying this from my first version design. Thanks.